sqlalchemy_exasol/base.py

```diff
@@ -1199,3 +1279,33 @@ def fkey_rec():
     def get_indexes(self, connection, table_name, schema=None, **kw):
         """EXASolution has no explicit indexes"""
         return []
+
+    def type_descriptor(self, typeobj):
```

Comment:
Did you try putting the ExaTimestamp in the `colspecs` mapping?

https://github.com/exasol/sqlalchemy-exasol/blob/master/sqlalchemy_exasol/websocket.py#L91